src/processor.cpp

```diff
@@ -0,0 +1,164 @@
+// This file is a part of Julia. License is MIT: https://julialang.org/license
+
+// Processor feature detection
+
+#include "julia.h"
+#include "julia_internal.h"
+
+extern "C" {
```

Comment:
ah okay. there won't be a C or C++ API somewhere in llvm or libuv for whatever you plan on using `/proc/cpuinfo` for?